epix/quantgen_wrapper.py

```diff
@@ -0,0 +1,183 @@
+import numpy as np
+import nestpy
+class BBF_quanta_generator:
+    def __init__(self):
+        self.er_par_dict = {
+            'W': 0.013509665661431896,
+            'Nex/Ni': 0.08237994367314523,
+            'py0': 0.12644250072199228,
+            'py1': 43.12392476032283,
+            'py2': -0.30564651066249543,
+            'py3': 0.937555814189728,
+            'py4': 0.5864910020458629,
+            'rf0': 0.029414125811261564,
+            'rf1': 0.2571929264699089,
+            'fano' : 0.059
+        }
+        self.nr_par_dict = {
+            "W": 0.01374615297291325, 
+            "alpha": 0.9376149722771664, 
+            "zeta": 0.0472,
+            "beta": 311.86846286764376, 
+            "gamma": 0.015772527423653895, 
+            "delta": 0.0620,
+            "kappa": 0.13762801393921467, 
+            "eta": 6.387273512457444, 
+            "lambda": 1.4102590741165675,
+            "fano" : 0.059
+        }
+        self.ERs = [7,8,11]
+        self.NRs = [0,1]
```

Comment:
@terliuk Is this a typo? Don't we have just `0` for NRs?

Reply:
1 corresponds to WIMP interaction type in NEST, so I added it there.